apps/web/src/styles.css

```diff
@@ -219,3 +215,33 @@ ul {
     background-position: 500%;
   }
 }
+
+@keyframes enter {
+  from{
+    opacity: 0;
+    transform: scale(95%);
+  }
+  to{
+    opacity: 1;
+    transform: scale(100%);
+  }
```

Comment:
Is it possible to use tailwind `@apply` directives?

Reply:
Done